client/src/components/landing/components/LandingTop.js

```diff
@@ -26,8 +27,17 @@ function LandingTop({ page }) {
   const showChineseLogos = ['chinese', 'chinese-tradition'].includes(
     clientLocale
   );
+
+  function localStorageCheck() {
```

Comment:
```suggestion
  function isFirstTime() {
```
This is a bit more descriptive: it tells the reader what the function does rather than how it does it.